### PR TITLE
Update z.json

### PR DESCRIPTION
**Pull request by zimmah**
added entry

### CODE REVIEW

File: mentors/javascript/z.json [hunk 1]
```diff
@@ -6,5 +6,13 @@
...
-  }
+  },
+  {
+  "github_username": "zimmah",
+  "name": "Zimri Leijen",
+  "link_text": null,
+  "link_url": null,
+  "avatar_url": null,
+  "bio": "Fullstack web developer using Javascript and React, cryptocurrency pioneer"
+}
```
Comment (rpottsoh) [inline mentors/javascript/z.json]:
```suggestion
    "github_username": "zimmah",
    "name": "Zimri Leijen",
    "link_text": null,
    "link_url": null,
    "avatar_url": null,
    "bio": "Fullstack web developer using Javascript and React, cryptocurrency pioneer"
  }
```

Just some minor formatting changes.

**Comment by rpottsoh** [comment]
@zimmah just a friendly reminder that this PR needs a little TLC from you.

**Comment by zimmah** [comment]
Sorry I somehow missed the notifications, thanks for reminding me.

**Comment by tejasbubane** [comment]
@zimmah Can you do these changes so we can get it merged?

**Comment by zimmah** [comment]
I made a new pull request because I think I deleted the branch where the original pull request originated from.

**Comment by tejasbubane** [comment]
@zimmah I cannot find the new pull request. Can you put the link here?

**Comment by zimmah** [comment]
hmm, i must be doing something wrong again. 😅

**Comment by SleeplessByte** [comment]
@zimmah are you still interested in this?

**Comment by SleeplessByte** [comment]
(Can't reopen the PR because the source repository has been deleted).

**Comment by zimmah** [comment]
i'm quite busy lately, i'll do it when I have more free time.

**Pull request closed without merging**